source/common/upstream/cluster_manager_impl.cc

```diff
@@ -575,7 +580,6 @@ bool ClusterManagerImpl::removeCluster(const std::string& cluster_name) {
     cm_stats_.cluster_removed_.inc();
     updateGauges();
     // Did we ever deliver merged updates for this cluster?
-    // No need to manually disable timers, this should take care of it.
     updates_map_.erase(cluster_name);
```

Comment:
Nit: I would remove the question in favor of a statement. Something like "Cancel any pending updates for this cluster."